tgui/packages/tgui/styles/interfaces/DecalPainter.scss

```diff
@@ -0,0 +1,8 @@
+.DecalPainter__CellButton {
+  margin: 2px !important;
+  padding: 0.5em !important;
```

Comment:
You can use `m={'2px'}` and `p={1}` properties on the Button to do the same thing. And not sure why did you use !important.

Reply:
I'd think so, but for some reason they don't from a class perspective. I'll double check if m and p work properly in this case - I originally was trying to do this with table and it was a mess until I sensibly switched to a Flex

Reply:
Button implements a Box (and passes all unhandled props to it), so yes, it supports all of these props.

Reply:
@stylemistake I didn't realize you could pass a string to `m` and `p` like that &mdash; neat. Problem with the DecalPainter__CellIcon one in the other syggestion: spritesheet css files make the icon elements `inline-block`, so the display has to be overriden. I've just made the icon div use `style={{display:'block'}}` since it's a single, simple property change.

Reply:
Ok, now I get why there was `display: block !important`. That kinda sucks.